src/animation/play_once_animation.rb

```diff
@@ -0,0 +1,22 @@
+require 'gosu'
+require 'src/animation'
+
+class PlayOnceAnimation < Animation
```

Comment:
That name sound a little bit stranger, what do you think about some thing like CompleteAnimation, ActionAnimation, InteractionAnimation ...? :nail_care: